wall_e/extensions/ban.py

```diff
@@ -113,12 +111,16 @@ async def intercept(self, guild: discord.Guild, member: Union[discord.User, disc
         while self.guild is None:
             await asyncio.sleep(2)
 
-        # need to read the audit log to grab mod, date, and reason
         self.logger.info(f"[Ban intercept()] guild ban detected and intercepted for user='{member}'")
-        self.logger.debug("[Ban intercept()] waiting 1 second to ensure ban log is created")
+
```

Comment:
is it possible to have this function just send a message back on the mod channel to instruct the mod who banned the user to just use the `.ban` command instead in an effort to cut down on the duplication of logic that seems to exist here?

Reply:
no, that's bad ux and sometimes its faster to ban someone the discord way. i dont care about the logic duplication it worth it for the convenient functionality.

Reply:
> no, that's bad ux 

Fair point, then can you update the class so that `.ban` and `intercept` intercept both call the same function for the actual wall_e banning logic aspect of their code?

> sometimes its faster to ban someone the discord way

What do you mean?

Reply:
> What do you mean? 

sometimes its faster to right click ban someone when its urgent, like when a bot account(bot compromised accounts) is spamming every channel in the discord. also for mods on mobile its faster to use the ui than to type out the command. 

> Fair point, then can you update the class so that .ban and intercept both call the same function for the actual wall_e banning logic aspect of their code?

Not purpose of this pr.

Reply:
> Not the purpose of this pr.

Maybe but the current state also is not best practice.

Reply:
and the necessary change is pretty easily done.